### DISCUSSION TITLE
NABox Appliance

### DISCUSSION THREAD

**Discussion by coreymr** [NetApp/harvest#753] [Q&A] [answered]
Hi, I'm new to this hub and looking at the discussions on upgrading Harvest.  Does NetApp support the OVA appliance for NABox anymore?  It sounds like there is an architectural redesign required if going to Harvest 2.x.  It would be nice if there was an NABox appliance to download or upgrade to instead (for those of us who aren't developers).

Anyone have insight/info on that?  Is this something in the works?

**Comment by cgrinds** [comment]
welcome @coreymr - I'm not sure what the status of nabox is. Perhaps @ybizeul can chime in? If you're comfortable with Docker, perhaps you can try the [Docker compose workflow](https://github.com/NetApp/harvest/tree/main/docker#harvest-grafana-prometheus-install). That is the workflow created and recommended by the Harvest dev team.

**Comment by ybizeul** [comment, marked as answer]
NAbox v3 is necessary to run Harvest 2, it is currently in beta but real close to GA so you can perfectly use it. It is available from the download page for NAbox !

**Reply by coreymr** [reply]
Is there an upgrade or do I install the new OVA appliance?  If that is the case, is there a data migration or is the v3 appliance completely new and we keep v2 for old perf data?

**Reply by coreymr** [reply]
@ybizeul I've imported the new 3.0 beta appliance and am having trouble with the basic configuration.  I'm not familiar with docker containers and can't seem to find the logs to troubleshoot the LDAP configuration and my first cluster harvest connection.
